test/e2e/autoscaling/cluster_autoscaler_scalability.go

```diff
@@ -59,7 +59,7 @@ type scaleUpTestConfig struct {
 	expectedResult *clusterPredicates
 }
 
-var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", func() {
+var _ = SIGDescribe("Cluster size autoscaler scalability [Slow]", func() {
```

Comment:
is `"[sig-autoscaling] ` the right label?

Reply:
https://github.com/kubernetes/kubernetes/blob/18605d88141df048af86650e18a4cc7aa5cd0b23/test/e2e/autoscaling/framework.go#L23

Reply:
it is

Reply:
nevermind, you added it in the commit message